test/unittest/unittest_nntrainer_layers.cpp

```diff
@@ -31,6 +31,18 @@
 using nntrainer::sharedConstTensor;
 using nntrainer::sharedTensor;
 
+static std::string getDimensionString(const nntrainer::TensorDim &dim) {
+  std::string dim_str;
+  for (unsigned int i = 0; i < nntrainer::MAXDIM; i++) {
+    dim_str += std::to_string(dim.getTensorDim(i));
+    dim_str += ",";
```

Comment:
Could you check if this is right?
```suggestion
    dim_str += ":";
```

Reply:
done.